domains/kate.json

```
@@ -0,0 +1,13 @@
+{
+    "description": "My Portfolio Website",
+    "repo": "https://github.com/kateshweta/portfolio_sk",
+    "owner": {
+    "username": "kateshweta",
+    "email": "kateshweta17@gmail.com"
+    },
+
+    "record": {
+    "CNAME": "https://kateshweta.github.io/portfolio_sk/"
```

CNAME can't contain protocol or `/`
```suggestion
    "CNAME": "kateshweta.github.io"
```